e2e/volume_ops_test.go

```diff
@@ -271,6 +272,39 @@ func testVolumeDelete(t *testing.T) {
 func testVolumeStart(t *testing.T) {
 	r := require.New(t)
 	r.Nil(client.VolumeStart(volname, false), "volume start failed")
+
+	vol, err := client.BricksStatus(volname)
+	for index := range vol {
+		r.NotZero(vol[index].Port)
+		r.True(vol[index].Online)
+	}
+
+	vol, err = client.BricksStatus(volname)
+	killBrick := vol[0].Info.Path
```

Comment:
This is just suggestion for this test to be more robust. You can randomise picking the brick:

```go
r := rand.New(rand.NewSource(time.Now().UnixNano()))
killBrick := vol[r.Intn(len(vol))].Info.Path
```

Reply:
Done